### PR TITLE
Feature/hcatoscea_test_data_ami_update

### PR DESCRIPTION
**Pull request by ami-day**
Fixed input test files (TEST01.xlsx, TEST02.xlsx) which were resulting in incorrect output files. Updated the arguments.csv file to reflect this.

### CODE REVIEW

File: hca2scea-backend/test/golden/TEST02/E-HCAD-36.idf.txt [hunk 1]
```diff
@@ -0,0 +1,30 @@
+
```
Comment (amnonkhen) [inline hca2scea-backend/test/golden/TEST02/E-HCAD-36.idf.txt]: I would like to suggest a differnt naming convention for the test data.
Please name the spreadsheet and the directory containing the idf and sdrf using the accession number.
In this case:
- spreadsheet: SRP274475.xlsx
- idf & sdrf should be in `test/golden/SRP274475`

**Review by amnonkhen — CHANGES_REQUESTED** [review]

File: hca2scea-backend/test/golden/TEST02/E-HCAD-36.idf.txt (hunk 1 above)
Comment (ami-day) [reply hca2scea-backend/test/golden/TEST02/E-HCAD-36.idf.txt]: Ok will do.

**Review by ami-day — COMMENTED** [review]
These changes are correct and intended.

**Comment by amnonkhen** [comment]
> These changes are correct and intended.

@ami-day I do not understand your comment.

**Comment by ami-day** [comment]
> > These changes are correct and intended.
> 
> @ami-day I do not understand your comment.

Hi @amnonkhen I'm not sure what I meant by this. I made the changes you suggested. Can this request be approved now?

**Review by amnonkhen — CHANGES_REQUESTED** [review]
Thanks for updating the hca to scea test input spreadsheet. I do not, however, see the [expected results files](https://github.com/ebi-ait/hca-to-scea-tools/tree/feature/hca-to-scea-test-data-ami-update/hca2scea-backend/test/golden), namely the idf/sdrf files for each of the test cases?
Before merging your changes, the tests should run successfully.
At least for your test cases.
You can run the tests on your environment by running python -munittest.
Also, once you push them to the remote branch github runs them for you. Please [examine the output](https://github.com/ebi-ait/hca-to-scea-tools/runs/5230981087?check_suite_focus=true), and fix the errors. If you don't undersatnd the error messages, talk to me.

**Comment by ami-day** [comment]
> Thanks for updating the hca to scea test input spreadsheet. I do not, however, see the [expected results files](https://github.com/ebi-ait/hca-to-scea-tools/tree/feature/hca-to-scea-test-data-ami-update/hca2scea-backend/test/golden), namely the idf/sdrf files for each of the test cases? Before merging your changes, the tests should run successfully. At least for your test cases. You can run the tests on your environment by running python -munittest. Also, once you push them to the remote branch github runs them for you. Please [examine the output](https://github.com/ebi-ait/hca-to-scea-tools/runs/5230981087?check_suite_focus=true), and fix the errors. If you don't undersatnd the error messages, talk to me.

Thanks @amnonkhen I have done this, the test is returning "OK" now.

File: hca2scea-backend/hca2scea.py [hunk 2]
```diff
@@ -202,6 +225,24 @@ def get_new_protocol_column_names(sdrf, counter):
 
     return new_column_names, counter
 
+def order_protocols(protocols_sdrf_before_sequencing):
+    collection_protocol_cols = []
+    dissociation_protocol_cols = []
+    enrichment_protocol_cols = []
+    library_preparation_protocol_cols = []
+    for col in list(protocols_sdrf_before_sequencing.columns):
+        if col.split(".protocol_core.protocol_id")[0] == "collection_protocol":
```
Comment (amnonkhen) [inline hca2scea-backend/hca2scea.py:234]: No need to repeat `col.split(".protocol_core.protocol_id")[0]` 4 times.
Define a variable and use it in the 4 locations.

File: hca2scea-backend/test/golden/output/SRP151260/E-HCAD-50.idf.txt [hunk 3]
```diff
@@ -11,10 +11,10 @@ Person Email																						alison.simmons@imm.ox.ac.uk
 Person Affiliation	MRC Weatherall Institute of Molecular Medicine (WIMM)	MRC Weatherall Institute of Molecular Medicine (WIMM)	MRC Weatherall Institute of Molecular Medicine (WIMM)	MRC Weatherall Institute of Molecular Medicine (WIMM)	MRC Weatherall Institute of Molecular Medicine (WIMM)	MRC Weatherall Institute of Molecular Medicine (WIMM)	Target Discovery Institute	MRC Weatherall Institute of Molecular Medicine (WIMM)	MRC Weatherall Institute of Molecular Medicine (WIMM)	John Radcliffe Hospital, University of Oxford	MRC Weatherall Institute of Molecular Medicine	Sir William Dunn School of Pathology, University of Oxford	MRC Weatherall Institute of Molecular Medicine	MRC Weatherall Institute of Molecular Medicine (WIMM)	MRC Weatherall Institute of Molecular Medicine (WIMM)	MRC Weatherall Institute of Molecular Medicine (WIMM)	MRC Weatherall Institute of Molecular Medicine (WIMM)	Target Discovery Institute	Target Discovery Institute	St Mary's Hospital, Imperial College	MRC Weatherall Institute of Molecular Medicine	MRC Weatherall Institute of Molecular Medicine (WIMM)	EMBL-EBI
```
Comment (amnonkhen) [inline hca2scea-backend/test/golden/output/SRP151260/E-HCAD-50.idf.txt:11]: How come we have Test01 output file but now Test01 spreadsheet in the arguments?
This file will not be used anywhere.

**Review by amnonkhen — CHANGES_REQUESTED** [review]

**Comment by ami-day** [comment]
Closing this branch, it is an older version that is now out of date.

**Pull request closed without merging**